jobs/managers/task.ts

```diff
@@ -128,7 +128,7 @@ export default async function refresh({ providers }: IReposJob): Promise<IReposJ
             console.log(
               `Update to corporate link: username changed from ${link.corporateUsername} to ${info.userPrincipalName}`
             );
-            link.corporateUsername = info.disuserPrincipalNameplayName;
+            link.corporateUsername = info.userPrincipalName;
```

Comment:
This appears to be a bug, but I went with the value based on the message above

Reply:
Agreed, thanks.